python_modules/dagster/dagster/_core/definitions/unresolved_asset_job_definition.py

```diff
@@ -48,6 +48,16 @@ class UnresolvedAssetJobDefinition(
         ],
     )
 ):
+    """Represents a job that targets an unresolved AssetSelection.
+
+    An unresolved job can't be executed directly, because it doesn't hold direct references to the
+    AssetsDefinition objects that it needs to execute.
+
+    An unresolved jobs is meant to be placed inside a Definitions object, alongside the selected
```

Comment:
```suggestion
    An unresolved job should be placed inside a Definitions object, alongside the selected
```

Suggesting 'should be' instead of 'meant to be' - is there something else one would do with this type of job?